city_scrapers/spiders/il_port_district.py

```diff
@@ -0,0 +1,181 @@
+                meetings_dates = row.xpath(".//div/text()").extract()
+                if not meetings_dates:
+                    continue
+            else:
+                meetings_dates = row
+
+            for i, date in enumerate(meetings_dates):
+                if not date:
+                    continue
+                special_meeting = None
+                if date.startswith("***") or date.endswith("***"):
+                    meeting_time = changed_meeting_time
+                elif date.startswith("**") or date.endswith("**"):
+                    special_meeting = "Special "
+                elif date.startswith("*") or date.endswith("*"):
+                    new_date = re.findall(r"(?<=\()(.*?)(?=NEW)", date)
+                    date = new_date[0][:-3] if new_date else date
+
+                date = date.strip(" *")
+
+                title = meeting_types[i] if not special_meeting\
+                    else special_meeting + meeting_types[i]
+
+                classification = self._parse_classification(i, meeting_types[i])
+
+                start = self._parse_start(year, date, meeting_time)
+
+                agendas_links = self._parse_agendas_links(classification)
+
+                minutes_links = None
+
+                if classification == "Board":
+                    minutes_links = self._parse_minutes_links(start)
+                links = agendas_links if not minutes_links else\
+                    agendas_links + minutes_links
+
+                meeting = Meeting(
+                    title=title,
+                    description="",
+                    classification=classification,
+                    start=start,
+                    end=None,
+                    all_day=False,
+                    time_notes="",
+                    location=self.location,
+                    links=links,
+                    source=response.url
+                )
+
+                meeting["status"] = self._get_status(meeting)
+                meeting["id"] = self._get_id(meeting)
+
+                yield meeting
+
+    def parse_agendas(self, response):
+        file_prefix = response.xpath("//tr/td[@class='views-field views-field-title']/text()")\
+            .extract()
+        file_prefix = [x.strip("\n ") for x in file_prefix]
+        file_links = response.xpath("//tr/td/a[@class='file-download']/@href").extract()
+        name_pattern = r"(?<=documents\/)(.*?)(?=.pdf)"
+        file_names = [re.findall(name_pattern, x)[0].replace('%', ' ') for x in file_links]
+
+        self.files_list = []
+
+        for link, prefix, name in list(zip(file_links, file_prefix, file_names)):
+            self.files_list.append({'title': prefix + " " + name, 'href': link})
+
+    def parse_minutes(self, response):
+        rows = response.xpath("//tr")
+        self.minutes_dict = {}
+        for row in rows:
+            file_name = row.xpath(".//td[@class='views-field views-field-title']/text()")\
```

Comment:
Similar to the agendas, if we know these are all minutes we don't need the meeting date associate since they're already included in the meeting details